### DISCUSSION TITLE
Yetiforce CRM URL

### DISCUSSION THREAD

**Discussion by LINNYAN86** [YetiForceCompany/YetiForceCRM#17317] [How to]
I would like to know i just forget to take a notes for Yetiforce CRM URL in my server and how to find out that. I would like to know how to get that information from the server.

Thanks to you all and we are looking for a solution for that.

Best regards,
Lin

**Comment by ghost** [comment]
After install the url will be in config/Main.php under

>        /** Backslash is required at the end of URL */
>         public static $site_URL = 'http://example.com/';
